invenio_app_ils/config.py

```diff
@@ -1003,6 +1078,44 @@ def _(x):
             ),
         ),
     ),
+    "ils-record-changes": dict(
+        cls=DateHistogramQuery,
+        permission_factory=backoffice_read_permission,
+        params=dict(
+            index="stats-ils-record-changes",
+            copy_fields=dict(
+                pid_type="pid_type",
+                method="method",
+            ),
+            required_filters=dict(
+                pid_type="pid_type",
+                method="method",
+            ),
+            metric_fields=dict(
+                count=("sum", "count", {}),
+            ),
+        ),
+    ),
+    "ils-record-changes-per-user": dict(
+        cls=ESTermsQuery,
+        permission_factory=superuser_permission,
```

Comment:
please add a recipe to out digital repositories on how to query this stat, make it detailed:)